File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.py
from sys import stdin
input = stdin.readline

N = int(input())
points = []

for _ in range(N):
    x, y = map(int, input().split())
    points.append((x,y))
        
sorted_points = sorted(points)

for point in sorted_points:
    print(point[0], point[1])
        

File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

const T = Number(input.shift());

const dot = input.map((line) => {
  const [x, y] = line.split(" ").map(Number);
  return { x, y };
});

dot.sort((a, b) => {
  if (a.x === b.x) {
    return a.y - b.y;
  }
  return a.x - b.x;
});

dot.forEach((dots) => {
  console.log(`${dots.x} ${dots.y}`);
});
